server/raft.go

```diff
@@ -3401,6 +3407,9 @@ func (n *raft) createCatchup(ae *appendEntry) string {
 	// Cleanup any old ones.
 	if n.catchup != nil && n.catchup.sub != nil {
 		n.unsubscribe(n.catchup.sub)
+	} else {
+		// Signal we've started catching up.
```

Comment:
"Signal to the upper layer that the following entries are catchup entries, up until the nil guard."